test/ui/test_spec.js

```diff
@@ -88,7 +88,9 @@ describe('UI tests', function() {
               });
           }, 10000);
         })
-        .finally(() => done());
+        .then(driver.executeScript("window.datalab.vminfo.vm_name='fake_vm_name_for_testing';" +
+              "require(['static/appbar'],appbar=>appbar.setAccountMenuVminfo());"))
```

Comment:
also, if we're on a VM, we shouldn't have the sign in button, so maybe we really should fool the container into thinking it's not running on a VM, just something to think about.